modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/snapshot/storage/SnapshotLocalFileSystemStorage.scala

```diff
@@ -50,6 +51,12 @@ final class SnapshotLocalFileSystemStorage[F[_]: Async: KryoSerializer, S <: Sna
   def read(hash: Hash): F[Option[Signed[S]]] =
     read(hash.coerce[String])
 
+  def readHash(ordinal: SnapshotOrdinal): F[Option[Hash]] =
+    read(ordinal).flatMap { test =>
+      val test2 = test.map(_.toHashed.map(_.hash)).sequence
+      test2
```

Comment:
what is `test2`? `map` + `sequence` = `traverse`

Reply:
`test2` was for debugging. It has been removed. This has been changed to

```scala
read(ordinal).flatMap(_.traverse(_.toHashed.map(_.hash)))
```